hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -96,77 +98,26 @@ def read_bmp(file):
     return data
 
 
-def get_neighbors(array, i, j, n=1):
-    i_max, j_max = array.shape
+def sum_valid_coherence_values(array: np.array, threshold: float = 0.3):
```

Comment:
We already specify the default value in `get_reference_pixel`.
```suggestion
def sum_valid_coherence_values(array: np.array, threshold: float):
```